lizrd/train/train_utils.py

```diff
@@ -68,6 +70,16 @@ def get_model(
     ).to(last_gpu)
 
     model = llm.LLM(embedding_layer, encoder_tower, head)
+
+    if load_weights_path:
+        print(f"Loading weights...")
+        assert os.path.exists(
+            load_weights_path
+        ), f"Path {load_weights_path} does not exist"
+        checkpoint = torch.load(load_weights_path)
+        model.load_state_dict(checkpoint["model"], strict=False)
+        print(f"Loaded from {load_weights_path}")
```

Comment:
1. Let wrap in into a function
2. Move it so it is next to `save_weights_path`

Reply:
done